### PR TITLE
Add a profile for stubruns in hasta

### PR DESCRIPTION
**Pull request by ramprasadn**
This PR adds a separate profile for -stub-runs in hasta cluster. Also, defines a sentieon specific parameter to run  raredisease pipeline with sentieon in hasta.

### CODE REVIEW

File: conf/hasta.config [hunk 1]
```diff
@@ -15,19 +15,35 @@ singularity {
...
     executor = 'slurm'
-    clusterOptions = { "-A $params.priority ${params.clusterOptions ?: ''}" }    
+    clusterOptions = { "-A $params.priority ${params.clusterOptions ?: ''}" }
+}
+
+params {
+    max_memory = 180.GB
+    max_cpus   = 36
+    max_time   = 336.h
 }
```
Comment (jemten) [inline conf/hasta.config:30]:
```suggestion
```

Comment (jemten) [reply conf/hasta.config:30]: Looks like a duplication of what row 15

File: conf/pipeline/raredisease/hasta.config [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
     }
     withName:'DEEPVARIANT' {
         cpus   = { check_max( 16      * task.attempt, 'cpus' ) }
         memory = { check_max( 80.GB   * task.attempt, 'memory' ) }
     }
-}
\ No newline at end of file
+    withName: BCFTOOLS_VIEW {
+        if (params.genome == 'GRCh37') {
+            ext.args = '--output-type v --apply-filters PASS --exclude "INFO/clinical_genomics_mipAF > 0.40 | INFO/swegenAF > 0.40 | INFO/clingen_ngiAF > 0.40 | INFO/gnomad_svAF > 0.40 "'
```
Comment (jemten) [inline conf/pipeline/raredisease/hasta.config:12]: Looks like the prior step outputs a gzipped vcf. Would be good to do this here as well.

**Review by jemten — COMMENTED** [review]
Hey @ramprasadn :)
Looks good and I really like the addition of the stub profile. Some comments

File: conf/pipeline/raredisease/hasta.config [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
         memory = { check_max( 80.GB   * task.attempt, 'memory' ) }
     }
-}
\ No newline at end of file
+    withLabel:'sentieon' {
+        beforeScript = { "export PATH=\$PATH:\$SENTIEON_INSTALL_DIR/sentieon-genomics-202010.02/bin" }
+    }
+    withName: BCFTOOLS_VIEW {
+        if (params.genome == 'GRCh37') {
+            ext.args = '--output-type z --apply-filters PASS --exclude "INFO/clinical_genomics_mipAF > 0.40 | INFO/swegenAF > 0.40 | INFO/clingen_ngiAF > 0.40 | INFO/gnomad_svAF > 0.40 "'
```
Comment (jemten) [inline conf/pipeline/raredisease/hasta.config:15]: What do you think about listing the args as a list instead? Like what's done [here](https://github.com/nf-core/rnaseq/blob/89bf536ce4faa98b4d50a8ec0a0343780bc62e0a/conf/modules.config#L500)

**Review by jemten — APPROVED** [review]
Looks good 😄 
Just a comment about the long args line. Feel free to implement it or not

**Comment by maxulysse** [comment]
You have conflicts in the hasta.config file

**Comment by jemten** [comment]
@ramprasadn, ready to merge?

**Pull request merged**